src/component/base/DraftEditorProps.js

```diff
@@ -17,6 +17,7 @@ import type {DraftBlockRenderMap} from 'DraftBlockRenderMap';
 import type {DraftDragType} from 'DraftDragType';
 import type {DraftEditorCommand} from 'DraftEditorCommand';
 import type {DraftTextAlignment} from 'DraftTextAlignment';
+import type {DraftTextDirectionality} from 'DraftTextDirectionality';
```

Comment:
fbjs already has `UnicodeBidiDirection`. I just gave it a quick test in your branch and it seems to work. You should be good to go without `DraftTextDirectionality`.

```js
import type {BidiDirection} from 'UnicodeBidiDirection';
```